solvers/cuml.py

```diff
@@ -78,4 +78,4 @@ def get_result(self):
             if self.lasso.fit_intercept:
                 coef = np.r_[coef, self.lasso.intercept_.to_numpy()]
 
-        return coef
+        return coef.astype(np.float64)
```

Comment:
it's for me the responsability of the objective class to make the conversions. So you avoid forgetting in a future solver

Reply:
that is more robust indeed !